### ISSUE TITLE
Allow for a learning rate parameter

### ISSUE THREAD

**Issue by PonteIneptique** [hipster-philology/pandora#78]
Right now, LR - at least for PyTorch - is hardcoded.
It should be possible to define it in the model-config level file or in the class initiation.

**Comment by emanjavacas** [comment]
There are many more params for the optimizer which could be added as well,
I didnt add any of them since I didnt see the option in the keras version
either

**Comment by PonteIneptique** [comment]
I could take of this one tbh so that we can configure SGD or Adam + Learning rate from the config file : 
https://github.com/hipster-philology/pandora/blob/master/pandora/impl/keras/model.py#L175-L179

**Comment by emanjavacas** [comment]
I guess you mean a different snippet, right? that one is about embeddings.
But yes, feel free to move learning rate to the config file.

**Comment by emanjavacas** [comment]
I've added #102 that adds an option for lr. I can extend it to also get an option for the optimizer type.

**Comment by PonteIneptique** [comment]
Fixed by #102
